thirdeye/pom.xml

```diff
@@ -543,6 +550,17 @@
         <artifactId>log4j-slf4j-impl</artifactId>
         <version>${log4j2.version}</version>
       </dependency>
+      <dependency>
+        <groupId>org.apache.commons</groupId>
+        <artifactId>commons-lang3</artifactId>
+        <version>${commons-lang3.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>org.apache.commons</groupId>
+        <artifactId>commons-math</artifactId>
+        <version>2.1</version>
+      </dependency>
```

Comment:
Wouldn't it be better to define this at the respective levels? These are not required across all the modules right!

Reply:
It is defined in this manner even in pinot. Didn't want to change the existing convention. As such I find maven to be extremely verbose. But I do agree and want to simplify the dependencies. I'm already working on that in a separate branch. But I'm keeping this unchanges in this PR.